Userland/Libraries/LibC/assert.cpp

```diff
@@ -13,8 +13,6 @@
 #include <syscall.h>
 #include <unistd.h>
 
-extern "C" {
-
```

Comment:
I don't follow the rationale of this commit.  We tend to put both the definitions and declarations of functions that might be called from C code inside `extern "C"` blocks.

Reply:
I'm OK, I'll change that then.